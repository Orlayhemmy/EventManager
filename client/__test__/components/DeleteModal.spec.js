import React from 'react';
import DeleteModal from '../../components/Modal/Container/DeleteModal';

describe('Delete modal', () => {
  const content = 'This is a delete modal';
  const wrapper = shallow(<DeleteModal content={content} />);

  it('should render the component without errors', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
