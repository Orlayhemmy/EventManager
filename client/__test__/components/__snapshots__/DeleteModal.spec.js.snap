// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Delete modal should render the component without errors 1`] = `
<div
  className="modal hide"
  id="deleteModal"
>
  <div
    className="modal-dialog mt-y4"
  >
    <div
      className="modal-content"
    >
      <div
        className="form-inner text-center"
      >
        This is a delete modal
      </div>
    </div>
  </div>
</div>
`;
